content/contact/index.md

```diff
@@ -30,9 +30,9 @@ We welcome vulnerability reports [according to our vulnerability disclosure poli
 
 ### Questions from the public and industry
 
-If you have a question that isn't on behalf of a U.S. government organization (such as if you're a member of the public or representing a company), we invite you to post your question [publicly as an issue on GitHub]({{site.github_url}}/issues/new) (requires a free GitHub account), so that we can write an answer available to everyone. If you prefer not to post publicly, you can email [{{site.email}}](mailto:{{site.email}}).
+If you have a question that isn't on behalf of a U.S. government organization (such as if you're a member of the public or representing a company), we invite you to post your question [publicly as an issue on GitHub](https://github.com/cloud-gov/site/issues/new) (requires a free GitHub account), so that we can write an answer available to everyone. If you prefer not to post publicly, you can email [{{site.email}}](mailto:{{site.email}}).
```

Comment:
```suggestion
If you have a question that isn't on behalf of a U.S. government organization (such as if you're a member of the public or representing a company), we invite you to post your question [publicly as an issue on GitHub]({{site.github_url}}/issues/new) (requires a free GitHub account), so that we can write an answer available to everyone. If you prefer not to post publicly, you can email [{{site.email}}](mailto:{{site.email}}).
```